iron-list.html

```diff
@@ -1390,11 +1390,14 @@
 
       forceUpdate = forceUpdate || this._scrollHeight === 0;
       forceUpdate = forceUpdate || this._scrollPosition >= this._estScrollHeight - this._physicalSize;
+      // Probably a bug that style.height is a string like '100px' and it is being compared to
+      // a number?
       forceUpdate = forceUpdate || this.grid && this.$.items.style.height < this._estScrollHeight;
 
       // Amortize height adjustment, so it won't trigger large repaints too often.
       if (forceUpdate || Math.abs(this._estScrollHeight - this._scrollHeight) >= this._optPhysicalSize) {
         this.$.items.style.height = this._estScrollHeight + 'px';
+        this.updateViewportBoundaries();
```

Comment:
Your example is interesting in that with Shadow DOM on there is no reason why the list would ever be 200px tall. 

To correct for that, you don't need to call anything as long as the following instructions are accurately fulfilled:

> Sizing iron-list
> 
> iron-list must either be explicitly sized, or delegate scrolling to an explicitly sized parent. By "explicitly sized", we mean it either has an explicit CSS height property set via a class or inline style, or else is sized by other layout means (e.g. the flex or fit classes).

Checkout an altered version of your sample here: http://jsfiddle.net/vLv0fdk8/ I've set the height in exact pixels as per your sample, but you could equally use `display: flexbox` or `postion: absolute` with top and bottom properties set, or some others that I can't think of right now for the same results.